elasticsearchIndexing/process-drafts.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+import logging
+import os
+import typing as t
+
+from utility import log
+from utility.create_config import create_config
+
+from elasticsearchIndexing.es_manager import ESManager
+from elasticsearchIndexing.models.es_indices import ESIndices
+
+
+def load_all_drafts(draft_dir: str) -> t.List[str]:
+    drafts = []
+    for filename in os.listdir(draft_dir):
+        if filename[-4:] == '.txt':  # if file name has .txt format
+            # Return only name of the draft
+            drafts.append(filename[:-4])
+    return drafts
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description='Process drafts in draft directory')
+    parser.add_argument('--config-path', type=str, default=os.environ['YANGCATALOG_CONFIG_PATH'],
+                        help='Set path to config file')
+    args = parser.parse_args()
+    config_path = args.config_path
+    config = create_config(config_path)
+    log_directory = config.get('Directory-Section', 'logs')
+    ietf_drafts_dir = config.get('Directory-Section', 'ietf-drafts')
```

Comment:
This loads files from `/var/yang/ietf/my-id-mirror`, but on the PROD we would like to load files from `/var/yang/ietf/my-id-mirror` - in this directory also archived drafts are stored
I suggest to load is_prod from config and if it is PROD, then index all the drafts from `/var/yang/ietf/my-id-mirror` and if not PROD (= is DEV) index from `/var/yang/ietf/my-id-mirror`